pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -183,10 +185,15 @@ func (a *Bootstrap) getTemplateData(installConfig *types.InstallConfig) (*bootst
 		PullSecret:   installConfig.PullSecret,
 		ReleaseImage: releaseImage,
 		EtcdCluster:  strings.Join(etcdEndpoints, ","),
+		Proxy:        installConfig.Proxy,
 	}, nil
 }
 
 func (a *Bootstrap) addStorageFiles(base string, uri string, templateData *bootstrapTemplateData) (err error) {
+	proxy := map[string]bool{
```

Comment:
the files will automatically be empty if proxy is not set, right?

Is it a problem if we write *empty* file to disk? If is not a problem, I think it will be cleaner to always write those file, even though they might be empty..?

Reply:
No issues were caused by having the files empty.  The cluster installed normally.  Updated the PR to remove this section.